TestApp/ViewController.swift

```diff
@@ -20,8 +20,8 @@ class ViewController: UIViewController {
         
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: TestApp/ViewController.swift;
  Line: 20;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)